src/wazuh_db/wdb_shared.c

```diff
@@ -25,6 +25,29 @@ const char* WDBC_RESULT[] = {
     [WDBC_UNKNOWN] = "unk"
 };
 
+/// Array of component strings.
+const char* wdbc_valid_components[] = { 
```

Comment:
Please, use the same trick as `WDBC_RESULT`, where each value is already tied to the enum value

Reply:
Done!